File: app/src/app/core/models/section.ts
/*
 * Este fichero forma parte de UBUMonitorWeb.
 *
 * @author Marcelo Verteramo Pérsico
 */

import { Module } from './module';

export type Section = {
  id: number;
  name: string | null;
  order: number | null;
  isVisible: boolean;
  isUserVisible: boolean;
  modules: Module[];
};

File: api/src/main/kotlin/es/ubu/lsi/ubumonitorweb/domain/Section.kt
/*
 * Este fichero forma parte de UBUMonitorWeb.
 *
 * @author Marcelo Verteramo Pérsico
 */

package es.ubu.lsi.ubumonitorweb.domain

data class Section(
  val id: Int,
  val name: String?,
  val order: Int?,
  val isVisible: Boolean,
  val isUserVisible: Boolean,
  val modules: List<Module>,
)
